realm/src/main/java/io/realm/internal/RealmCore.java

```diff
@@ -101,7 +101,7 @@ public static void gcGuaranteed(){
         Object obj = new Object();
         WeakReference<Object> ref = new WeakReference<Object>(obj);
         obj = null;
-        while(ref.get()!=null)
+        while (ref.get()!=null)
```

Comment:
Ha, never mind. I thought you were saying need a space in `ref.get()` like `ref.get( )`